django/db/migrations/writer.py

```diff
@@ -184,6 +185,9 @@ def as_string(self):
         if self.migration.replaces:
             items['replaces_str'] = "\n    replaces = %s\n" % self.serialize(self.migration.replaces)[0]
 
+        if self.migration.initial:
```

Comment:
I think this code is correct as is; I'm not sure what @markush is suggesting. Initial can be tri valued, but regardless we only ever want to write `initial = True` into auto-written migration files.